editor/src/core/performance/performance-regression-tests.spec.tsx

```diff
@@ -64,7 +64,7 @@ describe('React Render Count Tests -', () => {
 
     const renderCountAfter = renderResult.getNumberOfRenders()
     // if this breaks, GREAT NEWS but update the test please :)
-    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`404`)
+    expect(renderCountAfter - renderCountBefore).toMatchInlineSnapshot(`382`)
```

Comment:
This is unexpected

Reply:
Yeah, only came up after I merged master into the branch too, but I can't see anything that should've affected it.